paper-api/src/main/java/org/bukkit/inventory/Inventory.java

```diff
@@ -85,12 +85,11 @@ public interface Inventory extends Iterable<ItemStack> {
      * index of the varargs parameter. If all items are stored, it will return
      * an empty HashMap.
      * <p>
-     * If you pass in ItemStacks which exceed the maximum stack size for the
-     * Material, first they will be added to partial stacks where
-     * Material.getMaxStackSize() is not exceeded, up to
-     * Material.getMaxStackSize(). When there are no partial stacks left
-     * stacks will be split on Inventory.getMaxStackSize() allowing you to
-     * exceed the maximum stack size for that material.
+     * The maximum size of the stack added to this inventory will be the minimum
```

Comment:
Not the biggest fan of this wording.
Reads like any amount above said computed minimum is discarded?
When in reality, it will first be fit into partial stacks, respecting the existing items max stack size, and then splitting the leftover by its computed max size and adding it.

Reply:
perfect danke <3